CONTRIBUTING.md

```diff
@@ -47,97 +62,47 @@ Checking for Memory Leaks through GitHub
-- Carmichael Ong
-- Soha Pouya
-- Apoorva Rajagopal
-- Jeffrey A. Reinbolt
-- Ajay Seth
-- Amy Silder
-- Andreas Scholz
-- Michael A. Sherman
-- Ian Stavness
-- Katherine Steele
-- Justin Si
-- Darryl G. Thelen
-- Tom Uchida
-- Jack Wang
-- Kevin Xu
-- Jenny Yong
-
-# Coding Standards
-
-## Header Guards
+Coding Standards
+----------------
+- [Header guards](#header-guards)
+- [Creating new OpenSim objects](#creating-new-opensim-objects)
+- [Assignment operators in C++](#assignment-operators-in-c)
+- [Documenting your code](#documenting-your-code)
+- [Each line of text should be at most 80 characters](#each-line-of-text-should-be-at-most-80-characters)
+- [Replace tabs with four spaces](#replace-tabs-with-four-spaces)
+- [Renaming classes in the OpenSim API](#renaming-classes-in-the-opensim-api)
+- [Naming conventions](#naming-conventions)
+- [Throw and return are not functions](#throw-and-return-are-not-functions)
+- [Always use pre-increment and pre-decrement operators when you have a choice](#always-use-pre-increment-and-pre-decrement-operators-when-you-have-a-choice)
+- [Place pointer and reference symbols with the type](#place-pointer-and-reference-symbols-with-the-type)
+- [Removing methods](#removing-methods)
+
+### Header guards
 
 Header guards are preprocessor defines that surround every header file to prevent it from being included multiple times. OpenSim header guards should be written like this:
 
 ```cpp
-#ifndef OPENSIM_PROPERTY_TABLE_H_ // <-- yes, do it this way!
+#ifndef OPENSIM_PROPERTY_TABLE_H_
 #define OPENSIM_PROPERTY_TABLE_H_
  // ... stuff ...
 #endif // OPENSIM_PROPERTY_TABLE_H_
 ```
 This matches the scheme used in Simbody and avoids all of the following problems. (Trailing underscore, but not leading, is allowed.)
 
-They should not look like this:
-
-```cpp
-#ifndef _PropertyTable_h_ // <-- don't do it this way!
-#define _PropertyTable_h_
- // ... stuff ...
-#endif //_PropertyTable_h_
-```
-
-There are two problems with these header guards:
+Common problems with header guards:
 
 1. They can interfere with user code, and
 2. They violate the C++ standard.
 
-OpenSim uses many very common class names, like “Object” and “Array” and probably “PropertyTable”. These are likely to appear in other code libraries as well, so anyone who combines OpenSim API with other libraries or their own code may have conflicts. A simple rule to avoid conflicts is for the header guards to be unique symbols, easily achieved by including the product name in them (that is, they should contain “OPENSIM”).
+OpenSim uses many very common class names, like “Object” and “Array” and probably “PropertyTable”. These are likely to appear in other code libraries as well, so anyone who combines the OpenSim API with other libraries or their own code may have conflicts. A simple rule to avoid conflicts is for the header guards to be unique symbols, easily achieved by including the product name in them (that is, they should contain “OPENSIM”).
```

Comment:
The "probably" *probably* is an assessment of how common this class name is in other libraries. I think we should remove PropertyTable; the other two examples are sufficient to make the point.